completions/nush/eza.nu

```diff
@@ -28,12 +33,14 @@ export extern "eza" [
     --links(-H)                # List each file's number of hard links
     --inode(-i)                # List each file's inode number
     --blocksize(-S)            # List each file's size of allocated file system blocks
+    --time(-t) -d              # Which timestamp field to list
     --dereference(-X)          # dereference symlinks for file information
```

Comment:
nit: I know it's not from this PR but all the first letters are capitalized except for `dereference`